code/modules/mob/living/carbon/human/species/zombies.dm

```diff
@@ -37,17 +37,13 @@
 	language = "Zombie"
 	var/heal_rate = 1
 	var/regen_cooldown = 0
-	flesh_color = "#00FF00" // for green examine text
-
-/datum/species/zombie/infectious/handle_dna(mob/living/carbon/human/H, remove)
-	. = ..()
 
-/datum/species/zombie/infectious/spec_stun(mob/living/carbon/human/H,amount)
+/datum/species/zombie/infectious/spec_stun(mob/living/carbon/human/H, amount)
 	. = min(20, amount)
 
-/datum/species/zombie/infectious/spec_attacked_by(damage, damagetype = BRUTE, def_zone = null, blocked, mob/living/carbon/human/H)
+/datum/species/zombie/infectious/spec_apply_damage(damage = 0, damagetype = BRUTE, def_zone = null, blocked = 0, sharp = 0, obj/used_weapon = null)
```

Comment:
Ditto on overriding apply damage instead of a new proc unless there's a really good reason not to